Devscord.DiscordFramework/Framework/Server.cs

```diff
@@ -242,5 +245,11 @@ public static async Task<IEnumerable<string>> GetExistingInviteLinks(ulong serve
             var invites = await guild.GetInvitesAsync();
             return invites.Select(x => x.Url);
         }
+
+        private static Task BotDisconnected(Exception exception)
+        {
+            Log.Warning($"Bot disconnected {exception.Message} stack trace: {exception.StackTrace}");
+            return Task.CompletedTask;
```

Comment:
tutaj wrzuciłbym próbe automatycznego połączenia 
jakaś pętla która próbuje połączyć sie np 3 razy
i jak sie nie uda do wyrzuca wyjątek (wyłącza aplikacje + log)

Reply:
Właśnie sprawdzałem co się dzieje jak odłączy się od internetu i z tego co zaobserwowałem, to bot automatycznie się wznawia po jakimś czasie kiedy uzyska dostęp do sieci
Na razie myślę, że tak można zostawić, a na produkcji zobaczyć jak to wygląda, bo teraz będziemy mieć zapisywanie Disconnected do logów

Reply:
dobry pomysł